# Question

Will the code work? Why or why not?

```js
const getUsers = async () => {
  const url = 'https://jsonplaceholder.typicode.com/users';
  const users = await fetch(url).then(r => r.json()).catch((err) => []);
  console.log(users)
  return users;
}
getUsers()
```

Again, running the code gives you the answer, so try to figure it out on your own first!

# Answer
This should work because it was getting the result from a normal `.then()` `.catch()` promise and the `async/await` allows it too be put into the variable that can  be used later on. Plus the function called at the end.

# Teacher Feedback